### PR TITLE
Improve operator documentation

### PR DESCRIPTION
**Pull request by L3o-pold**
- Fix deadlockInterval for ingester.
- Add index-prefix option for es storage

### CODE REVIEW

File: content/docs/1.17/operator.md [hunk 1]
```diff
@@ -263,12 +263,13 @@ spec:
...
           brokers: my-cluster-kafka-brokers.kafka:9092
       ingester:
-        deadlockInterval: 5 # <2>
+        deadlockInterval: 5s # <2>
   storage:
     type: elasticsearch
     options:
       es:
         server-urls: http://elasticsearch:9200
+        #index-prefix: my-prefix
```
Comment (annanay25) [inline content/docs/1.17/operator.md]: Why comment? When published on the main site it looks like its leftover by mistake

File: content/docs/1.17/operator.md [hunk 2]
```diff
@@ -263,12 +263,13 @@ spec:
...
           brokers: my-cluster-kafka-brokers.kafka:9092
       ingester:
-        deadlockInterval: 5 # <2>
+        deadlockInterval: 5s # <2>
   storage:
     type: elasticsearch
     options:
       es:
         server-urls: http://elasticsearch:9200
+        index-prefix: my-prefix
```
Comment (objectiser) [inline content/docs/1.17/operator.md]: Not sure why this `index-prefix` parameter should be included here? The example is primarily about streaming, not the storage. So think this one should be removed.

Comment (L3o-pold) [reply content/docs/1.17/operator.md]: I moved this to the External Elasticsearch part.

**Comment by objectiser** [comment]
@L3o-pold Thanks!

**Comment by objectiser** [comment]
Sorry, I merged prematurely - the changes should also be made to the `next-release` folder.

@L3o-pold If you don't mind creating a PR to apply the changes there - otherwise I will do it later.

**Comment by L3o-pold** [comment]
@objectiser see https://github.com/jaegertracing/documentation/pull/379

**Pull request merged**